subprojects/libft/src/ft_getline.c

```diff
@@ -118,27 +118,26 @@ static int	get_next_line_with_state(const int fd, char **line, char **fd_state)
 ** fd_state_store if necessary.
 */
 
-static int	get_next_line_c(const int fd, char **line, t_list **state)
```

Comment:
just gonna pretend i believe this works,
tho i dont understand why you do not not use the t_list_conn struct

Reply:
duplicate of above review item